### ISSUE TITLE
First Blood Dota 2 dApp

### ISSUE THREAD

**Issue by CryptoEEra** [Firstbloodio/dapp#2]
I just completed all the boxes necessary to setup my player and witness accounts on your dApp, but there is no any other button like "Save" or "Continue" or "Sign up" etc... to allow me to go further and now I'm stuck. Any help please or am I just wasting my time? I attached a screenshot below so you can take a look, I'm on Windows:
![firstblood 2018-10-26 15 01 06](https://user-images.githubusercontent.com/36078535/47597776-274c0900-d947-11e8-8e9e-348d68ad78d1.png)

**Comment by zackcoburn** [comment]
If you scroll all the way to the bottom of the FirstBlood Dapp Disclaimer, the "Please scroll..." message should be replaced with a checkbox and a button. 

 * Are you sure you are scrolled all the way to the very bottom of the FirstBlood Dapp Disclaimer?
 * It looks like your screen is shrunk down horizontally and vertically. What is your screen resolution, and is the Dapp maximized?
 * Which version of Windows are you using?

**Comment by CryptoEEra** [comment]
Yes, I scrolled all the way down, but still no button. I’m on Windows 10.

**Comment by zackcoburn** [comment]
Hi, please see the latest release, which should fix this issue: https://github.com/Firstbloodio/dapp/releases/tag/v1.0.2

**Comment by zackcoburn** [comment]
Are you able to login if you leave the Witness part blank?

**Comment by CryptoEEra** [comment]
I tried to submit just the Player, no Witness -- and this time says
"There's no such player username available."
But that's impossible. I copied and pasted directly from Steam.  Besides, I
have 2 Steam accounts and I tried with both and it says the same: "There's
no such player username available."

**Comment by zackcoburn** [comment]
When you typed in your Steam username, did you use the [USERNAME] part of https://steamcommunity.com/id/[USERNAME], and does that URL work for your username?

**Comment by CryptoEEra** [comment]
Btw, I pressed the question mark next to the box and I got:
"What should I enter?
Enter your Steam Community URL. This will be the username that you use to
play matches."
So, I must enter an URL, not my actual Steam Username?

**Comment by zackcoburn** [comment]
You should enter your username, but if you go to the URL https://steamcommunity.com/id/[USERNAME], that page should work (does it? -- if not, you need to mark your account as public).

**Comment by CryptoEEra** [comment]
I attached below a screenshot with what I got once I went to the link with
my username:

**Comment by zackcoburn** [comment]
I don't see the screenshot?

**Comment by CryptoEEra** [comment]
I sent you 3 times already the Screenshot attached and it says "delivered
successfully". Still can't see it?

**Comment by zackcoburn** [comment]
Hmm, I still can't see it. Maybe upload to imgur.com and post a link?

**Comment by CryptoEEra** [comment]
Basically, this is what is on the page:
Error
Sorry!

An error was encountered while processing your request:

The specified profile could not be found.

Here's a link to the Steam Community home page <https://steamcommunity.com/>
.

[image: Valve Logo]© Valve Corporation. All rights reserved. All trademarks
are property of their respective owners in the US and other countries.
Some geospatial data on this website is provided by geonames.org
<https://steamcommunity.com/linkfilter/?url=http://www.geonames.org>.
Privacy Policy <http://store.steampowered.com/privacy_agreement/>   |  Legal
<https://store.steampowered.com/legal/>  |  Steam Subscriber Agreement
<http://store.steampowered.com/subscriber_agreement/>

**Comment by zackcoburn** [comment]
Ah, I see. You should follow this guide first, to make your account public: https://support.steampowered.com/kb_article.php?ref=4113-YUDH-6401

**Comment by CryptoEEra** [comment]
My account is Public long ago since I had to set up to be able to play on
your Online platform: firstblood.io  But anyway, I double checked now and
yes -- it was Public all along.

**Comment by zackcoburn** [comment]
Does the profile (URL) username match your Steam username?

**Comment by CryptoEEra** [comment]
I'm confused already. What profile (URL) and what Steam username?

**Comment by zackcoburn** [comment]
![image](https://user-images.githubusercontent.com/15804989/47724890-8ea6d900-dc25-11e8-99c5-39816fd4f438.png)

**Comment by CryptoEEra** [comment]
Profile name is: ch921111
But the box next to http://steamcommunity.com/id/ is blank.

**Comment by zackcoburn** [comment]
Put ch921111 in the blank box and save it. (Then the URL should work.)

**Comment by CryptoEEra** [comment]
So, it allowed me to finally create an account; but only without witness.
Does it mean I can’t become a witness anymore?

**Comment by zackcoburn** [comment]
I'm working on a fix for the "can't become a witness" problem on Windows 10. Will release a new version soon.

**Comment by CryptoEEra** [comment]
Thanks, appreciate your effort; keep me updated please.

**Comment by zackcoburn** [comment]
Check out the latest release (1.0.3). This should take care of the issues with logging in as a witness.

**Comment by ghost** [comment]
So, what’s going on? Are you going to make available for the Windows users
to be able to become witnesses?

**Comment by zackcoburn** [comment]
Not sure if you saw my previous reply (I edited it):

> Check out the latest release (1.0.3). This should take care of the issues with logging in as a witness.

**Comment by ghost** [comment]
May I get the link please?

**Comment by zackcoburn** [comment]
https://github.com/Firstbloodio/dapp/releases/tag/v1.0.3

(And, in general, all new releases will appear here: https://github.com/Firstbloodio/dapp/releases)
